app/src/main/java/com/example/plantsapp/presentation/ui/notification/TaskNotificationManager.kt

```diff
@@ -22,12 +28,24 @@ class TaskNotificationManager @Inject constructor(
     @ApplicationContext private val context: Context
 ) {
     private var notificationId = 0
```

Comment:
Lets leave here "//TODO"  comment
Idea is to avoid such hardcoding and incrementing and make notificationId depend on task it need to show 
We will think on it later, cause this thing currently is working